include/class.thread.php

```diff
@@ -1476,6 +1476,22 @@ static function fromFormattedText($text, $format=false) {
             return new ThreadBody($text);
         }
     }
+
+    static function clean($text, $format=null) {
```

Comment:
Shouldn't this replace `getClean`? The `ThreadEntry::create()` method uses `getClean` to determine if the entry is empty.